packages/tocco-ui/src/Pagination/Pagination.js

```diff
@@ -11,15 +11,14 @@ import {StyledPagination, StyledPaginationButton} from './StyledComponents'
  */
 const Pagination = ({totalCount, recordsPerPage, currentPage, onPageChange, intl}) => {
   const totalPages = Math.ceil(totalCount / recordsPerPage)
-  const start = (currentPage - 1) * recordsPerPage + 1
-  const to = currentPage * recordsPerPage
+  const start = Math.min((currentPage - 1) * recordsPerPage + 1, totalCount)
```

Comment:
Math.min for start necessary?

Reply:
not anymore, since we don't show the info text when there is no content. But before it was saying `1 bis 0 von 0` because `start` was at least `1`.